### PR TITLE
Show Ayah Reference when Arabic is not Shown

### PR DESCRIPTION
**Pull request by DoozyDoz**
In response to issue #1963

before changes
Screenshot_20220702_103151
after changes
Screenshot_20220702_103123

### CODE REVIEW

**Comment by ahmedre** [comment]
wa3laikum alsalam,
jazakumAllah khairan - however, the sura string is already there, just under the arabic - by adding it here we'd have it multiple times which would be too much. we moved it there to support cases where multiple translations are shown, and thus showing this multiple times wouldn't be ideal.

**Comment by DoozyDoz** [comment]
> wa3laikum alsalam, jazakumAllah khairan - however, the sura string is already there, just under the arabic - by adding it here we'd have it multiple times which would be too much. we moved it there to support cases where multiple translations are shown, and thus showing this multiple times wouldn't be ideal.

Salam alaikum brother
I noticed too, its only abscent when the option to not show ayah is unchecked in setting( ie. If only translation is shown)
So I guess I should check the option before appending the surah reference

File: app/src/main/java/com/quran/labs/androidquran/util/ShareUtil.kt [hunk 1]
```diff
@@ -77,6 +78,12 @@ class ShareUtil @Inject internal constructor(private val quranDisplayData: Quran
           append(text)
         }
       }
+      val arabicIsShownBeforeTranslation = QuranSettings.getInstance(context).wantArabicInTranslationView()
+      if (!arabicIsShownBeforeTranslation){
```
Comment (ahmedre) [inline app/src/main/java/com/quran/labs/androidquran/util/ShareUtil.kt:82]:
```suggestion
      if (ayahInfo.arabicText == null){
```

**Comment by ahmedre** [comment]
jazakumAllah khairan - will merge this shortly in sha' Allah.

**Comment by ahmedre** [comment]
jazakumAllah khairan

**Pull request merged**